support/hololens/ServoApp/ServoControl/Servo.cpp

```diff
@@ -69,6 +82,9 @@ Servo::Servo(hstring url, GLsizei width, GLsizei height, float dpi,
   o.enable_subpixel_text_antialiasing = false;
   o.vr_pointer = NULL;
 
+  // Example of how to specify platform specific logger
+  // o.ptrfunc = NULL; //&myptrfunc;
```

Comment:
We want the logger initialized for our application, so no need for any examples - let's create our logging function and initialize it here.